docs/gettingstarted/modfiles.md

```diff
@@ -81,67 +46,63 @@ modId = "examplemod1"
+| `mandatory`    | boolean | **mandatory** |                                     Whether the game should crash when this dependency is not met.                                      | `mandatory=true`                             |
+| `versionRange` | string  |     `""`      |  The acceptable version range of the language loader, expressed as a [Maven Version Range][mvr]. An empty string matches any version.   | `"versionRange=[1, 2)"`                      |
+| `ordering`     | string  |   `"NONE"`    | Defines if the mod must load before (`"BEFORE"`) or after (`"AFTER"`) this dependency. If the ordering does not matter, return `"NONE"` | `ordering="AFTER"`                           |
+| `side`         | string  |   `"BOTH"`    |                    The [physical side][dist] the dependency must be present on: `"CLIENT"`, `"SERVER"`, or `"BOTH"`.                    | `side="CLIENT"`                              |
+| `referralUrl`  | string  |   *nothing*   |                                     A URL to the download page of the dependency. Currently unused.                                     | `referralUrl="https://library.example.com/"` |
 
 :::danger
-The `ordering` of two mods may cause a crash due to a cyclic dependency: for example, mod A must load `"BEFORE"` mod B and mod B `"BEFORE"` mod A.
+The `ordering` of two mods may cause a crash due to a cyclic dependency, for example if mod A must load `"BEFORE"` mod B and at the same time, mod B must load `"BEFORE"` mod A.
 :::
 
 Mod Entrypoints
 ---------------
 
-Now that the `mods.toml` is filled out, we need to provide an entrypoint to being programming the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `mods.toml`.
+Now that the `mods.toml` is filled out, we need to provide an entrypoint for the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `mods.toml`.
 
 ### `javafml` and `@Mod`
 
-`javafml` is a language loader provided by Forge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `mods.toml`. From there, all initialization logic (e.g., [registering events][events], [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class. The mod bus can be obtained from `FMLJavaModLoadingContext`.
+`javafml` is a language loader provided by NeoForge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `mods.toml`. From there, all initialization logic (e.g. [registering events][events] or [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class.
```

Comment:
It might be worth mentioning that there must be an exact 1-to-1 mapping between mod IDs in the mods.toml and mod IDs in `@Mod` annotations